src/Tribe/Template/Month.php

```diff
@@ -895,6 +895,14 @@ private function get_daily_events( $date ) {
 				), $this->args
 			);
 
+			// If the request is false or not set we assume the request is for all events, not just featured ones.
+			if (
+				tribe_is_truthy( tribe_get_request_var( 'featured', false ) )
+				|| tribe_is_truthy( tribe_get_request_var( 'tribe_featuredevent', false ) )
+			) {
+				$args['featured'] = true;
```

Comment:
Do you not want to set it to false? (I see that empty is the same, but wonder if there is benefit to setting false)
To set either way you could do:
`$args['featured'] = tribe_is_truthy( tribe_get_request_var( 'featured', false ) ) || tribe_is_truthy( tribe_get_request_var( 'tribe_featuredevent', false ) );`

Reply:
Yeah, the thing is the following (I left a comment regarding this in the PR description). From what I talked with Luca, in ORM, if:

- the featured arg is true, then it fetches only featured events.
- the featured arg is false, then it fetches not featured events.
- not defined, fetches all.

That's why we have it inside the if, because if the query var is not there, we shouldn't be adding the `$args['featured']` with false.

Reply:
👍